tests/test_session.py

```diff
@@ -21,6 +21,18 @@ def test_pubkey_auth(self):
             self.user, self.user_key, publickey=self.user_pub_key,
             passphrase=''), 0)
 
+    def test_failed_password_auth(self):
+        self.assertRaises(
+            AuthenticationError,
+            self.session.userauth_password('FAKE USER', 'FAKE PASSWORD')
```

Comment:
Function and arguments should be passed in separately - `assertRaises` calls the function.

Should be:

```
         self.assertRaises(
            AuthenticationError,
            self.session.userauth_password, 'FAKE USER', 'FAKE PASSWORD'
         )
```

Reply:
done